core/include/traccc/finding/finding_algorithm.ipp

```diff
@@ -100,10 +104,18 @@ finding_algorithm<stepper_t, navigator_t>::operator()(
         const unsigned int previous_step =
             (step == 0) ? std::numeric_limits<unsigned int>::max() : step - 1;
 
+        for (unsigned int i_seed = 0; i_seed < seeds.size(); i_seed++)
+            n_trks_per_seed[i_seed] = 0;
```

Comment:
One line will be enough:
`std::fill(n_trks_per_seed.begin(), n_trks_per_seed.end(), 0);`